desktop/electron/main.ts

```diff
@@ -101,6 +101,12 @@ app && app.whenReady().then(() => {
     }
   })
 
+  // handler for textToAudio function
```

Comment:
I think using api / IAPI  would be cleaner as it gives automatic typescript support and reduces boilerplate. You can see examples in routes.ts